### PR TITLE
Update to 1.17.1

### PR DESCRIPTION
**Pull request by pixeils**

### CODE REVIEW

**Comment by Fallen-Breath** [comment]
Why are there tons of indent changes

File: utils/config.py [hunk 1]
```diff
@@ -41,107 +37,4 @@
 		"Fallen_Breath",
 		"Steve"
 	]
-}
```
Comment (Fallen-Breath) [inline utils/config.py:44]: What's this deletion for

Comment (pixeils) [reply utils/config.py:44]: i was having an issue with it giving me invalid username so i wasted a bunch of time going over random stuff before i realised i just needed to use email, anything not directly related is an artefact of me testing updated files from pycraft or something, anything not directly related to the packet stuff is just me being dumb

**Comment by pixeils** [comment]
okay im just going to make a new fork to fix git history, and hence a new pr in a bit

**Pull request closed without merging**